### PR TITLE
Add missing package

### PR DESCRIPTION
**Pull request by ani4aniket**
## Summary
Add the missing `he@1.2.0` package, which was accidentally removed [here](https://github.com/facebook/react-native-website/pull/2129/commits/f38f62f680043b1b65b9043e8eb40085b4a7ef62#diff-07f3a4647460e74f9514d8def137963aL49).

This package was responsible for encoding string which is being used [here](https://github.com/facebook/react-native-website/blob/6e41a04f987b8161baaa578bb3334857e4896348/website/scripts/sync-api-docs/propFormatter.js#L38).

## Changes
Add the missing package `he@1.2.0` as devDependency.

## Changelog
[Internal]

### CODE REVIEW

**Comment by motiz88** [comment]
@ani4aniket Could you please update the description to link to the previous PRs, explain where we use this package, etc? (Mainly for the benefit of someone looking at the history of the repo in the future.)

File: website/package.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
   },
   "dependencies": {
     "docusaurus": "1.14.6",
+    "he": "^1.2.0",
```
Comment (motiz88) [inline website/package.json:41]: This can be in `devDependencies` like the other doc generator dependencies.

File: website/yarn.lock [hunk 2]
```diff
@@ -9,7 +9,7 @@
   dependencies:
     "@babel/highlight" "^7.0.0"
 
-"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.10.4", "@babel/code-frame@^7.8.3":
+"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.10.4":
```
Comment (motiz88) [inline website/yarn.lock:12]: It's pretty odd that this changed. I would expect only `he` to be added to the lockfile, with no other changes. Does running `yarn` on master (without this PR) produce these extra changes as well?

Comment (ani4aniket) [reply website/yarn.lock:12]: Yes. I have just pulled the updated master and then added `he`.

Comment (ani4aniket) [reply website/yarn.lock:12]: @motiz88 I have added `he` as devDependency in the latest commit https://github.com/facebook/react-native-website/pull/2143/commits/547db4e82f649f42181c74d199f9a2aec9688135.

Comment (motiz88) [reply website/yarn.lock:12]: Hey @ani4aniket, have you had a chance to check whether running `yarn` on master _before this PR_ produces changes to `yarn.lock`? If so, those should be fixed in a separate commit prior to this PR.

Comment (ani4aniket) [reply website/yarn.lock:12]: Sure, will do that.

Comment (ani4aniket) [reply website/yarn.lock:12]: @motiz88 Yes running `yarn` on master is producing the changes to `yarn.lock`. I've tested this on the updated master.

Screenshot 2020-08-20 at 2 51 52 PM

Comment (Simek) [reply website/yarn.lock:12]: @motiz88 This `yarn.lock` change is fine, it's the last know to me side effect of the merge of earlier auto-gen related PRs (which was not fixed yet).

**Comment by Simek** [comment]
@motiz88 What's the status of this PR?

**Comment by Simek** [comment]
Closing in favor of #2323 which added the missing package to the `sync-api-docs` workspace.

**Pull request closed without merging**